tests/functional/test_config_groups.py

```diff
@@ -328,11 +328,101 @@ def test_delete_host_from_group_after_delete_in_component(
     def test_delete_host_from_group_after_it_deleted(self, sdk_client_fs, provider):
         """Test that host removed from provider conf group after deleting"""
 
-        with allure.step("Create config group for provider and add host"):
-            test_host = provider.host_create(fqdn=FIRST_HOST)
-            provider_group = provider.group_config_create(name=FIRST_GROUP)
-            provider_group.host_add(test_host)
-            assert_host_is_in_group(provider_group, test_host)
+        test_host = provider.host_create(fqdn=FIRST_HOST)
+        provider_group = _create_group_and_add_host(provider, test_host)
+        assert_host_is_in_group(provider_group, test_host)
         with allure.step("Delete host"):
             test_host.delete()
         self.check_no_hosts_in_group(provider_group)
+
+
+class TestChangeGroupsConfig:
+    """Tests for changing group config"""
+
+    ASSERT_TYPE = ["float", "boolean", "integer", "string", "list", "file", "option", "text", "structure", "map"]
+
+    PARAMS_TO_CHANGE = {
+        "attr": {},
+        "config": {
+            "float": 1.1,
+            "boolean": False,
+            "integer": 0,
+            "password": "123",
+            "string": "string2",
+            "list": ["/dev/rdisk0s4", "/dev/rdisk0s5", "/dev/rdisk0s6"],
+            "file": "file content test",
+            "option": "WEEKLY",
+            "text": "testtext",
+            "structure": [{"code": 3, "country": "Test1"}, {"code": 4, "country": "Test2"}],
+            "map": {"age": "20", "name": "Chloe", "sex": "f"},
+        },
+    }
+
+    @allure.step("Check group config has been changed")
+    def _check_changed_values_in_group(
+        self, values_after: OrderedDict, values_before: OrderedDict, expected_values: dict = PARAMS_TO_CHANGE["config"]
+    ):
+        """Checks that params in config group are equal to expected and password has been changed"""
+
+        for item in self.ASSERT_TYPE:
+            assert (
+                values_after[item] == expected_values[item]
+            ), f'Value is "{values_after[item]}", but should be {expected_values[item]}'
+            assert values_after["password"] != values_before["password"], "Password has not changed"
```

Comment:
can you add `password` to `ASSERT_TYPE`?

Reply:
I can't test the value of the password